src/math/mod.rs

```diff
@@ -31,12 +32,8 @@ mod truncf;
 //mod service;
 
 pub use self::{
-    ceilf::ceilf, expf::expf, fabs::fabs, fabsf::fabsf, floor::floor, floorf::floorf, fmodf::fmodf,
-    hypot::hypot, hypotf::hypotf, log2::log2, log2f::log2f, logf::logf, powf::powf, round::round,
```

Comment:
the log2 and log2f imports got removed here. Maybe due to some rebase? Please add them back

Reply:
ah that was it thanks